doctor.py

```diff
@@ -346,6 +347,43 @@ def check_duplicate_5_tuples(self):
             
         return not bool(bad)
 
+    def check_connection_distribution(self):
+        """Checking if connections are unevenly distributed across workers
+
+        Usually, connections should be distributed evenly across workers. If connections are
+        unevenly distributed, load balancing might be not working properly.
+        """
+
+        files = find_recent_log_files(self.log_directory, "conn.*", days=1)
+        if not files:
+            self.err("No conn log files in the past day???")
+            return False
+
+        nodes = defaultdict(int)
+        for rec in read_bro_logs_with_line_limit(reversed(files), 10000):
+            # Only count connections that have completed a three way handshake
```

Comment:
Probably want to remove the if statements here.. those are mostly useful for checking for missed_bytes and for SAD/had connections.  For testing load balancing, would likely want to look at all connections regardless of if they look broken or not.